hirola/front/templates/front/confirmation_email_body.html

```diff
@@ -0,0 +1,86 @@
+                <div class="title">
+                        <div class="col-a">
+                            <h2>teke</h2>
+                        </div>
+                        <div class="col-b">
+                                <h2>Order Confirmation</h2>
+                        </div>
+                </div>
+                <div class="content">
+                        <p>
+                            Thanks for shopping at teke {{ user.first_name }}.
+                        </p>
+                        <p>
+                            We will update you when the item ships.
+                        </p>
+                </div>
+        </div>
+        <div class="row">
+                <div class="title">
+                        <div class="col-a">
+                                <h2>Order Details</h2>
+                        </div>
+                </div>
+                <div class="content">
+                        {% for obj in cart %}
+                        <li><span class="text-title">{{obj.phone_model_item}}</span> Ksh. {{obj.phone_model_item.price}}</li>
+                        {% endfor %}
+                        <p>
+                            <span class="text-title" id="total">Order Total: Ksh. </span>{{cart_total}}
+                        </p>
+                </div>
+        </div>
+        <div class="row">
+                <div class="title">
+                        <div class="col-a">
+                            <h2>Shipping Details</h2>
+                        </div>
+                </div>
+                <div class="content">
+                        <p>
+                            These items will ship in 2-3 weeks. We will update you on the delivery
+                            progress as the items are shipped.
+                        </p>
+                        {% if shipping_address != None %}
+                        <p>
+                            <span class="text-title">Phone Number:</span> {{shipping_address.phone_number}}
+                        </p>
+                        <p>
+                            <span class="text-title">Town:</span> {{shipping_address}}
+                        </p>
+                        {% else %}
+                        <p>
+                            <span class="text-title">Phone Number:</span> {{user.phone_number}}
+                        </p>
+                        {% endif %}
+                </div>
+        </div>
+        <div class="row">
+                <div class="content">
+                        <p>
+                           Have a question? please visit <span><a href="{{ protocol }}://{{ domain }}{% url 'front:help' %}">teke help center</a></span>
+                        </p>
+                        <p>
+                           We hope to see you soon again.
+                        </p>
+                        <p>
+                            <span><a href="peter.tekesquared.com">teketeke.com</a></span>
```

Comment:
Hey, by hardcoding this information to the href for peter.tekesquared.com means that this will not work when the site is merged to ragz.tekesquared.com or tekesquared.com. The site is also not teketeke.com. I'd suggest using a similar implementation like https://github.com/JamesKirkAndSpock/Hirola/blob/83ea48996a0605eae5bea0d7662e9b014bd12fd3/hirola/front/templates/front/confirmation_email_body.html#L81